README.md

```diff
@@ -106,27 +107,73 @@ _This is also how you would use this module without `@decorator` syntax, althoug
 
 ### Usage with React Hooks
 
-Following the example above, once at least one component is wrapped with `@track()` we can access a `tracking` object via the `useTracking` hook from anywhere in the sub-tree:
+Following the example above, we can also access the `trackEvent` method via the `useTracking` hook from anywhere in the tree:
 
 ```js
 import { useTracking } from 'react-tracking';
 
-const SomeChild = () => {
-  const tracking = useTracking();
+const FooPage = () => {
+  const { trackEvent } = useTracking({ page: 'FooPage' });
```

Comment:
Let's use the `<Track />` component here, even though it's not actually necessary, just to illustrate the intended usage as the first example in the docs.